### PR TITLE
Add logging to get information about refresh token version

### PR DESCRIPTION
**Pull request by cYKatherine**

### CODE REVIEW

File: src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java [hunk 1]
```diff
@@ -9,17 +9,21 @@
...
+import java.util.Map;
 
 public class EncryptedTokenEncoder implements ITokenEncoder {
-
     private final KeyManager keyManager;
 
     public EncryptedTokenEncoder(KeyManager keyManager) {
         this.keyManager = keyManager;
     }
+    private final Map<String, Counter> _refreshTokenVersionCounters = new HashMap<>();
```
Comment (jon8787) [inline src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java:26]: sure we need a map? some recent changes to Operator removed maps like this one

Comment (cYKatherine) [reply src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java:26]: This is to make sure we don't create duplicated metrics on the same site id

Comment (jon8787) [reply src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java:26]: but micrometer already ensures this right?

Comment (vishalegbert-ttd) [reply src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java:26]: Hey Jonathan, is there an example of this? A lot of our metrics use a dictionary to keep track of how various labels map to different counters

Comment (jon8787) [reply src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java:26]: Let me try to find. @mcollins-ttd recently removed one of these dictionaries from memory

File: src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java [hunk 2]
```diff
@@ -259,13 +263,24 @@ public AdvertisingToken decodeAdvertisingTokenV3orV4(Buffer b, byte[] bytes, Tok
         );
     }
 
+    private void recordRefreshTokenVersionCount(String siteId, TokenVersion tokenVersion) {
+        _refreshTokenVersionCounters.computeIfAbsent(siteId, id -> Counter
+                .builder(String.format("uid2_refresh_token_%s_served_count", tokenVersion.toString().toLowerCase()))
```
Comment (vishalegbert-ttd) [inline src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java:268]: Now that I think about it.. version should be a label as well to make it easier to visualise the metric across versions. But we also don't update refresh version often so I'll leave it up to you

Comment (vishalegbert-ttd) [reply src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java:268]: So we'll need to map <version, site ID> to counter instead

**Review by vishalegbert-ttd — APPROVED** [review]

File: src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java (hunk 2 above)
Comment (jon8787) [reply src/main/java/com/uid2/operator/service/EncryptedTokenEncoder.java:268]: agree, definitely better to use label instead of a dynamic metric name

**Pull request merged**